routes/images.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const router = express.Router()
+const debug = require('debug')('k8s-autodeploy:services')
+const {executeCommand, deployment} = require('../util/commands')
+const {commonErrorHandler} = require('../util/errors');
+
+// Restart all deployments that run a specific image
+router.post('/:images/rollout', async function (request, response, next) {
+  const dockerTag = request.body.push_data.tag
+  const images = request.params.images.split(',')
+
+  debug('NEW rollout request. Images => %s, DockerTag => %s', request.params.images, dockerTag)
+  try {
+    await Promise.all(
+      images.map(async image => {
+        // Gets all deployments running the given image separated by space
+        const command = `kubectl get deployments -o jsonpath="{range .items[*]}{.metadata.name}{' '}{.spec.template.spec.containers[*].image}{'\\n'}{end}" | grep ${image} | awk '{print $1}' | xargs`
```

Comment:
When I tried this command locally I had to provide the `-A` flag to `kubectl`. Otherwise I would get `No resources found in default namespace.`. Was this not the case for you?

Reply:
I think the intention is the k8s-autodeploy is run in the same namespace as the services, so it will only be able to list pods in that namespace.

Also, to me the use of `xargs` seems a bit weird, is it just to have everything on the same line? (You could also use `awk '{ print $1 }' ORS=' '`, but xargs is cool too!).

Reply:
Leaving it just for the current namespace for now